GVFS/GVFS.Mount/InProcessMount.cs

```diff
@@ -141,7 +141,8 @@ public void Mount(EventLevel verbosity, Keywords keywords)
                         // Use TracingConstants.MessageKey.InfoMessage rather than TracingConstants.MessageKey.CriticalMessage
                         // as this message should not appear as an error
                         { TracingConstants.MessageKey.InfoMessage, "Virtual repo is ready" },
-                    });
+                    },
+                    Keywords.Telemetry);

```

Comment:
Should this still be `EventLevel.Critical`, or can we make this Info level? 

CC: @sanoursa. Saeed, do you remember if we perviously got push back for Critical + Telemetry events?

Reply:
I discussed this with @sanoursa offline and he agreed this should no longer be `Critical`.

Reply:
Thanks @wilbaker , I updated as suggested